### PR TITLE
Checkpoint 1 for Issue 1: Travis runs all tests under Python 2.7 with coverage

### PR DESCRIPTION
**Pull request by jamadden**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+language: python
+sudo: false
+python:
+  - 2.7
+#  - 3.4
+#  - 3.5
+#  - 3.6
+#  - pypy-5.4.1
+script:
+  - coverage run `which zope-testrunner` --test-path=src  --auto-color --auto-progress
```
Comment (mgedmin) [inline .travis.yml]: I usually prefer `coverage run -m zope.testrunner`, but this works fine too.

File: setup.py [hunk 2]
```diff
@@ -21,70 +21,80 @@
...
+        + '\n\n' +
+        read('src', 'zope', 'file', 'upload.txt')
+        + '\n\n' +
+        read('src', 'zope', 'file', 'contenttype.txt')
+        + '\n\n' +
+        read('src', 'zope', 'file', 'browser.txt')
+        + '\n\n' +
+        read('CHANGES.txt')
+    ),
+    keywords = "zope3 web html ui file pattern",
```
Comment (mgedmin) [inline setup.py]: Drop spaces around `=`

File: setup.py [hunk 3]
```diff
@@ -21,70 +21,80 @@
...
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Environment :: Web Environment',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Zope Public License',
+        'Programming Language :: Python',
+        'Programming Language :: Python:: 2 :: Only',
+        'Programming Language :: Python:: 2.7',
+        'Programming Language :: Python :: Implementation :: CPython',
```
Comment (mgedmin) [inline setup.py]: So you've tested this on pypy2 and found it didn't work?

File: setup.py [hunk 4]
```diff
@@ -21,70 +21,80 @@
...
+        'Natural Language :: English',
+        'Operating System :: OS Independent',
+        'Topic :: Internet :: WWW/HTTP',
+        'Framework :: Zope3'
+    ],
+    url='http://cheeseshop.python.org/pypi/zope.file',
+    license='ZPL 2.1',
+    packages=find_packages('src'),
+    package_dir = {'': 'src'},
```
Comment (mgedmin) [inline setup.py]: Drop spaces around `=`

File: setup.py [hunk 5]
```diff
@@ -21,70 +21,80 @@
...
+        'zope.interface',
+        'zope.location',
+        'zope.mimetype>=2.1',
+        'zope.publisher',
+        'zope.schema',
+        'zope.security',
+        'zope.size',
+    ],
+    include_package_data = True,
+    zip_safe = False,
```
Comment (mgedmin) [inline setup.py]: Spaces around `=`

File: .travis.yml (hunk 1 above)
Comment (jamadden) [inline .travis.yml]: Oh cool, I didn't realize that was an option.

File: setup.py (hunk 2 above)
Comment (jamadden) [inline setup.py]: Thanks! I caught several of those, but clearly not all.

File: setup.py (hunk 3 above)
Comment (jamadden) [inline setup.py]: Sadly yes. https://travis-ci.org/zopefoundation/zope.file/jobs/222233352

File: setup.py (hunk 2 above)
Comment (mgedmin) [inline setup.py]: flake8 editor integration (e.g. https://github.com/w0rp/ale for vim) FTW! ;)

**Pull request merged**